onmt/modules/multi_headed_attn.py

```diff
@@ -175,8 +211,25 @@ def forward(self, key: Tensor, value: Tensor,
                 query, key, value = self.linear_query(query),\
                                     self.linear_keys(query),\
                                     self.linear_values(query)
+                query = shape(query, self.dim_per_head)
                 key = shape(key, self.dim_per_head)
                 value = shape(value, self.dim_per_head)
+
+                if self.max_relative_positions == -1:  # Rotary Embeddings
```

Comment:
Can we think of a better way to hide the feature from the users? :)

Reply:
Or maybe this -1 is set in another part of the code that I missed?

Reply:
I need to find an elegant way to make rotary an option.